sfge/src/main/java/com/salesforce/config/EnvUtil.java

```diff
@@ -6,8 +6,8 @@
 public final class EnvUtil {
     private static final String ENV_RULE_THREAD_COUNT = "SFGE_RULE_THREAD_COUNT";
     private static final String ENV_RULE_THREAD_TIMEOUT = "SFGE_RULE_THREAD_TIMEOUT";
-    private static final String ENV_RULE_ENABLE_WARNING_VIOLATION =
-            "SFGE_RULE_ENABLE_WARNING_VIOLATION";
+    private static final String ENV_RULE_DISABLE_WARNING_VIOLATION =
```

Comment:
I renamed the SFGE env-var so its name exactly matched the SFCA env-var, but this necessitated renaming a few methods and inverting a few booleans.
If we'd prefer to not change the name in SFGE, and instead just invert the value in `SfgeWrapper.ts`, we can do that instead.